recipe/build.sh

```diff
@@ -11,13 +11,17 @@ source activate "${CONDA_DEFAULT_ENV}"
 if [ "$(uname)" == "Darwin" ]
 then
     export CXX="${CXX} -stdlib=libc++"
+    export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib
+    export LIBRARY_PATH="${PREFIX}/lib"
```

Comment:
Oh this last line should be for both platforms. In particular, we need this for Linux.